articles/quickstart.md

```diff
@@ -486,3 +486,44 @@ Init:One  0s=490  1s=510  agree=1000
 ```
 
 Our statistics for the first qubit haven't changed (50-50 chance of a 0 or a 1), but now when we measure the second qubit, it is __always__ the same as what we measured for the first qubit. Our `CNOT` has entangled the two qubits, so that whatever happens to one of them, happens to the other. If you reversed the measurements (did the second qubit before the first), the same thing would happen. The first measurement would be random and the second would be in lock step with whatever was discovered for the first.
+
+
+## Estimating Resources
+
+Some times researchers are interested not in the actual execution of the quantum program, but need to get an estimation
```

Comment:
```suggestion
Sometimes researchers are interested not in the actual execution of the quantum program, but need to get an estimation
```

Reply:
I would also clarify that this is typically not because the execution result is not interesting, but because it requires more qubits than can be classically simulated, so you can't get the result.